osama-breakout-room/Shirin Muataz/Style.css

```diff
@@ -0,0 +1,148 @@
+* {
+  padding: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  box-shadow: 0px 0px 5px 0px grey;
+  padding: 6px;
+}
+
+.wrapper {
+  width: 800px;
+  margin: 0 auto;
+}
+
+.logo {
+  float: left;
+}
+
+.logo img {
+  width: 50px;
+  height: auto;
+}
+
+nav {
+  float: right;
+}
+
+.btn {
+  display: inline-block;
+  border-radius: 12px;
+  padding: 6px 8px;
+  margin: 6px;
+  text-align: center;
+  font-size: 14px;
+}
+
+a.btn {
+  text-decoration: none;
+}
+
+.btn-bordered {
+  border: 1px solid #1da1f2;
+  color: #1da1f2;
+}
+
+.btn-blue {
+  border: 1px solid #1da1f2;
+  background-color: #1da1f2;
+  color: white;
+}
+
+.profile-cover-container {
+  width: 100%;
+  height: 300px;
+}
+
+.profile-image-container {
+  width: 150px;
+  height: 150px;
+  overflow: hidden;
+  margin: -75px 0 -75px 50px;
+}
+
+.profile-image-container img {
+  width: 100%;
+  height: 150px;
+  border-radius: 50%;
+  border: 2px solid white;
+}
+
+.profile-info-container {
+  margin: 20px 0 20px 220px;
+}
+
+#name {
+  margin-bottom: 8px;
+}
+
+.profile-buttons-container {
+  text-align: right;
+  margin-top: -24px;
+}
+
+#location {
```

Comment:
Try to stick to classes when it comes to styling your webpage unless you really need to use an id.